lib/rules/no-private-routing-service.js

```diff
@@ -55,7 +60,19 @@ module.exports = {
         });
 
         if (hasRoutingServiceDecorator) {
-          context.report({ node, message: ERROR_MESSAGE });
+          context.report({ node, message: PRIVATE_ROUTING_SERVICE_ERROR_MESSAGE });
+        }
+      },
+
+      Literal(node) {
+        if (typeof node.value === 'string' && node.value.includes(ROUTER_MICROLIB_NAME)) {
```

Comment:
What would be an example of an okay use of the string `_routerMicrolib`?

Reply:
While I agree it would be difficult to find a valid use case for that string/property name, it's still not great practice to globally disallow it. We try to scope our rules to specific Ember code as much as possible. That said, I'm willing to proceed with this change, as long as we move it behind a new rule option as suggested in my other comment.